### DISCUSSION TITLE
I am not able to center align the text

### DISCUSSION THREAD

**Discussion by bodamithun** [SixLabors/ImageSharp.Drawing#214] [Q&A] [answered]
I am writing text on an image, I am unable to align it correctly based on the horizontal alignment.. I am setting the textOption like below... but the text not aligning correctly with HorizontalAlignment .
```c#
 TextOptions textOptions =  new TextOptions(scaledFont)
            {
                HorizontalAlignment = horizontalAlignment.Center,
                VerticalAlignment = VerticalAlignment.Top,
                WrappingLength = targetWidth, 
                Origin = new PointF(xCordinate, yCordinate)
            }; 
```
Can somebody please point me what exactly I am missing.

**Comment by JimBobSquarePants** [comment, marked as answer]
You haven’t provided enough information for us to help I’m afraid.

What are your expectations vs the output?

**Reply by JimBobSquarePants** [reply]
It's very likely that you're simply using the wrong property. 

[HorizontalAlignment](https://docs.sixlabors.com/api/Fonts/SixLabors.Fonts.TextOptions.html#SixLabors_Fonts_TextOptions_HorizontalAlignment) sets the alignment of the text box.

[TextAlignment](https://docs.sixlabors.com/api/Fonts/SixLabors.Fonts.TextOptions.html#SixLabors_Fonts_TextOptions_TextAlignment) sets the alignment of the text within the box.
